api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -16,301 +16,282 @@
-        final List<RunSchedule> loadRunSchedule = runScheduleManager
-                .loadAllSchedulesBySchedulableId(RUN_ID, ScheduleType.PIPELINE_RUN);
-        assertEquals(0, loadRunSchedule.size());
-        Mockito.verify(runScheduler, Mockito.times(2)).unscheduleRunSchedule(any());
+        doReturn(Optional.of(createRunSchedule(1L, RUN_ID, RunScheduledAction.PAUSE, CRON_EXPRESSION1)))
+                .when(mockRunScheduleDao).loadRunSchedule(1L);
+        doReturn(Optional.of(createRunSchedule(2L, RUN_ID, RunScheduledAction.RESUME, CRON_EXPRESSION2)))
+                .when(mockRunScheduleDao).loadRunSchedule(2L);
+
+        runScheduleManager.deleteSchedules(RUN_ID, ScheduleType.PIPELINE_RUN, Arrays.asList(1L, 2L));
+
+        verify(mockRunScheduler, times(2)).unscheduleRunSchedule(any());
     }
 
     @Test
-    @WithMockUser(username = USER_OWNER)
     public void testDeleteAllRunSchedules() {
-        runScheduleManager.createSchedules(RUN_ID, ScheduleType.PIPELINE_RUN,
-                        Arrays.asList(testRunScheduleVO, testRunScheduleVO2));
-        Mockito.verify(runScheduler, Mockito.times(2)).scheduleRunSchedule(any());
+        doReturn(
+                Arrays.asList(
+                        createRunSchedule(1L, RUN_ID, RunScheduledAction.PAUSE, CRON_EXPRESSION1),
+                        createRunSchedule(2L, RUN_ID, RunScheduledAction.RESUME, CRON_EXPRESSION2)))
+                .when(mockRunScheduleDao).loadAllRunSchedulesBySchedulableIdAndType(RUN_ID, ScheduleType.PIPELINE_RUN);
 
         runScheduleManager.deleteSchedules(RUN_ID, ScheduleType.PIPELINE_RUN);
-        final List<RunSchedule> loadRunSchedule = runScheduleManager
-                .loadAllSchedulesBySchedulableId(RUN_ID, ScheduleType.PIPELINE_RUN);
-        assertEquals(0, loadRunSchedule.size());
-        Mockito.verify(runScheduler, Mockito.times(2)).unscheduleRunSchedule(any());
 
+        verify(mockRunScheduleDao).deleteRunSchedules(anyLong(), any());
+        verify(mockRunScheduler, times(2)).unscheduleRunSchedule(any());
     }
 
     @Test
-    @WithMockUser(username = USER_OWNER)
     public void testDeleteRunSchedulesForPipeline() {
-        runScheduleManager.createSchedules(RUN_ID, ScheduleType.PIPELINE_RUN,
-                Arrays.asList(testRunScheduleVO, testRunScheduleVO2));
-        Mockito.verify(runScheduler, Mockito.times(2)).scheduleRunSchedule(any());
+        doReturn(
+                Arrays.asList(
+                        createRunSchedule(1L, RUN_ID, RunScheduledAction.PAUSE, CRON_EXPRESSION1),
+                        createRunSchedule(2L, RUN_ID, RunScheduledAction.RESUME, CRON_EXPRESSION2)))
+                .when(mockRunScheduleDao).loadAllRunSchedulesBySchedulableIdAndType(RUN_ID, ScheduleType.PIPELINE_RUN);
 
-        runScheduleManager.deleteSchedulesForRunByPipeline(testPipeline.getId());
+        doReturn(Arrays.asList(createPipelineRun(RUN_ID, testPipeline.getId())))
```

Comment:
It's a nice idea to use `Collections.singletonList` rather than `Arrays.asList` for singleton lists.